omod/src/main/webapp/pages/login.gsp

```diff
@@ -175,7 +175,7 @@
                                     onclick="javascript:window.location = '/${ contextPath }/logout'"
                                     value="${ ui.message("general.cancel") }" />&nbsp;&nbsp;
                             <% } %>
-                                <input id="loginButton" class="btn ${ ui.message(selectLocation ? "confirm" : "btn-success") }" type="submit"
+                                <input id="loginButton" class="confirm" type="submit"
```

Comment:
Did you intentionally drop `btn` from class?

Reply:
yes i did

Reply:
Why did you?

Reply:
i wanted to use the class used in the [style guide](https://github.com/openmrs/openmrs-module-uicommons/blob/849890ec565dcabe340ad1e030fac66bd928bc9d/omod/src/main/webapp/pages/styleGuide.gsp#L1204) for the login button of the ui common scripts that are run

Reply:
Was it the cause of the reported problem and hence leading to your removing it?

Reply:
yes, the login button had lost its color, the confirm class which displays the color was conditioned to activate if the `selectLocation` variable is true  but this doesnt happen even when you select a location while logging in else btn-success which does have a background color

Reply:
Does it fail to work if you restored it to its original value? https://github.com/openmrs/openmrs-module-referenceapplication/commit/4ec2330d9c55093b1d9bec7d239e5e7f4ac9ef39#diff-8400366cbe867ce06a94434c16300f70f098b1487d29e1eea8f970cb25fda8d3L171

Reply:
it doesnt work with the original value because  `btn btn-success` nolonger exists but instead its `btn-success ` ..Am thinking these css changes were as a result of the bootstrap that was recently introduced as the foundation for reff app in the openmrs UI

Reply:
Can't you fix it without losing the logic which determines the class basing on `selectLocation`?

Reply:
sure i have done a fix here